pkg/policy/l4.go

```diff
@@ -43,6 +44,88 @@ type L4Filter struct {
 	Ingress bool
 }
 
+// FillPortRuleHTTP fills in the http port rule fields
+// as a regex into a single string
+func FillPortRuleHTTP(httpRule []api.PortRuleHTTP) []AuxRule {
+	l7rules := []AuxRule{}
+	for _, h := range httpRule {
+		r := AuxRule{}
```

Comment:
Set the L7 protocol / type / parser here:
```
r := AuxRule{L7Parser: "http"}
```

Reply:
done